test/TabExpansion.Tests.ps1

```diff
@@ -240,7 +240,7 @@ Describe 'TabExpansion Tests' {
             $alias = "test-$(New-Guid)"
 
             &$gitbin config alias.$alias checkout
-            (&$gitbin config --get-all alias.$alias).Count | Should Be 1
+            @(&$gitbin config --get-all alias.$alias).Count | Should Be 1
```

Comment:
This is not needed e.g. `(1,2).count` returns `2` , `(1).count` returns `1` and `($null).count` returns `0`.

Reply:
I'll go ahead and revert this.  (The thing I was seeing was that it was resolving the expression as a single string and not an array...)